content/en/docs/reference/using-api/cel.md

```diff
@@ -0,0 +1,330 @@
+---
+title: Common Expression Language in Kubernetes
+reviewers:
+- jpbetz
+- cici37
+content_type: concept
+weight: 35
+min-kubernetes-server-version: 1.25
+---
+
+<!-- overview -->
+
+The [Common Expression Language (CEL)](https://github.com/google/cel-go) is used
+in the Kubernetes API to declare validation rules, policy rules, and other
+constraints or conditions.
+
+CEL expressions are evaluated directly in the
+{{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}, making CEL a
+convenient alternative to out-of-process mechanisms, such as webhooks, for many
+extensibility use cases. Your CEL expressions continue to execute so long as the
```

Comment:
`as long as`?

Reply:
AFAIK, both "so long as" and "as long as" are considered idiomatic.